### PR TITLE
Fix feed qty

### PR DESCRIPTION
**Pull request by mbmcgarry**
I removed the original function for the PR (NonFissileMultiplier) and wrote a new general function, multi_mass_frac.  It takes in a map of Nuc and returns their combined mass / total material mass.

### CODE REVIEW

**Comment by scopatz** [comment]
This fails CI

**Comment by rwcarlsen** [comment]
But not because of anything in this PR - it sems to be a problem related to pyne data and/or pyne decay functionality.

File: src/toolkit/mat_query.h [hunk 1]
```diff
@@ -35,6 +35,9 @@ class MatQuery {
   /// Returns the mass fraction of nuclide nuc in the material.
   double mass_frac(std::string nuc);
 
+  /// Returns the mass fraction of nuclide nuc in the material.
+  double multi_mass_frac(std::set<Nuc> nucs);
```
Comment (gidden) [inline src/toolkit/mat_query.h]: I think I would prefer this to simply overload `mass_frac`, as the "multi" part is implied in the function argument. Do others have an opinion?

**Comment by mbmcgarry** [comment]
Says it failed, but the details do not show any failures. CI problem?

**Comment by scopatz** [comment]
> Says it failed, but the details do not show any failures. CI problem?

Probably an issue communicating back to gh. Try relaunching.

**Comment by scopatz** [comment]
http://cyclus-ci.fuelcycle.org/dashboard btw

File: src/toolkit/mat_query.h (hunk 1 above)
Comment (scopatz) [inline src/toolkit/mat_query.h]: I agree with @gidden

File: src/toolkit/mat_query.h [hunk 2]
```diff
@@ -35,6 +35,9 @@ class MatQuery {
   /// Returns the mass fraction of nuclide nuc in the material.
   double mass_frac(std::string nuc);
 
+  /// Returns the mass fraction of nuclide nuc in the material.
```
Comment (scopatz) [inline src/toolkit/mat_query.h]: This docstring needs an update :)

Comment (mbmcgarry) [inline src/toolkit/mat_query.h]: oops!

On Mar 12, 2015, at 9:10 AM, Anthony Scopatz <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/toolkit/mat_query.h<https://github.com/cyclus/cyclus/pull/1093#discussion_r26303611>:

> @@ -35,6 +35,9 @@ class MatQuery {
>    /// Returns the mass fraction of nuclide nuc in the material.
>    double mass_frac(std::string nuc);
>
> +  /// Returns the mass fraction of nuclide nuc in the material.


This docstring needs an update :)

**Comment by scopatz** [comment]
Thanks @mbmcgarry - this is a beautiful example of a minimal PR.

**Comment by mbmcgarry** [comment]
Thanks!

On Mar 12, 2015, at 2:03 PM, Anthony Scopatz <notifications@github.com<mailto:notifications@github.com>> wrote:


Merged #1093<https://github.com/cyclus/cyclus/pull/1093>.

**Pull request merged**